collection-scripts/gather_resources

```diff
@@ -23,6 +23,9 @@ resources+=(istag)
 resources+=(nodes)
 resources+=(machines)
 
+# Storage
+resources+=(StorageClass CSIDriver CSINode CSIStorageCapacity VolumeAttachment PersistentVolume)
```

Comment:
Nit: all the resource type in this script are in lowercase, plural format. WDYT about changing into:

```suggestion
resources+=(storageclasses csidrivers csinodes csistoragecapacities volumeattachments persistentvolumes)
```